README.md

```diff
@@ -57,6 +57,18 @@ As the Postgresql docker container has its 5432 port forwarded on the local mach
 
 To access the database using the pgAdmin you have to fill in also the container IP beside the database names and access credentials. The container IP can be seen with `docker ps`.
 
+### Migrations
+
+For testing and documentation purposes we have db migrations.
+
+We use [`postgrator`](https://github.com/rickbergfalk/postgrator) and you can find the sql files in the [`database/migrations`](/database/migrations) folder.
+
+To run the migrations you just need to execute
+
+`node database/migrate.js --version=<version>`
```

Comment:
no, you have to specify the version, I don't think `postgrator` works without the version number, but I haven't check